src/containers/Collection/CollectionList/CollectionList.tsx

```diff
@@ -300,7 +297,9 @@ export const CollectionList = () => {
             navigate(`/${groups ? 'group/' : ''}collection/add`);
           },
         }}
-        filters={filter}
+        filters={{
+          groupType: groups ? WA_GROUPS_COLLECTION : CONTACTS_COLLECTION,
+        }}
```

Comment:
Let's not change this and keep the filters as a state variable. 
filters is used as a dependency array in one of the useEffect inside List component that calls the fetch and count API. If you define it this way then on each re-render it creates a new object with this group attribute which in turn triggers the useEffect and calls the list API again.

You can check this while adding a new group to a collection. If you search for a waGroup it calls the list and count api for groups again and again